build.sbt

```diff
@@ -1,6 +1,9 @@
 lazy val `http4s-jdk-http-client` = project.in(file("."))
   .settings(commonSettings, releaseSettings, skipOnPublishSettings)
-  .settings(crossScalaVersions := Nil)
+  .settings(
+    crossScalaVersions := Nil,
+    mimaFailOnNoPrevious := false,
```

Comment:
This doesn't mix in `mimaSettings`, and since we don't publish the root, I think we don't want to.  I'm not sure the cleanest way to handle this, but this works.

Reply:
`.disablePlugins(MimaPlugin)` should work too.

Reply:
I like that a lot better. Thanks!